models/settings_helpers_test.go

```diff
@@ -63,6 +66,9 @@ func TestSettings(t *testing.T) {
 			DataRetention: 30 * 24 * time.Hour,
 			AWSPartitions: []string{"aws"},
 		}
+		expected.SaaS.STTCheckIntervals.StandardInterval = 24 * time.Hour
```

Comment:
let's include them as fields of `expected` like `MetricsResolutions.HR`

Reply:
@BupycHuk `SaaS` is embedded anonymous structure, so if do it like you said the result will be:
```
		expected := &models.Settings{
			MetricsResolutions: models.MetricsResolutions{
				HR: 5 * time.Second,
				MR: 10 * time.Second,
				LR: time.Minute,
			},
			DataRetention: 30 * 24 * time.Hour,
			AWSPartitions: []string{"aws"},
			SaaS: struct {
				// Percona Platform user email
				Email string `json:"email"`
				// Percona Platform session Id
				SessionID string `json:"session_id"`
				// Security Threat Tool enabled
				STTEnabled bool `json:"stt_enabled"`
				// List of disabled STT checks
				DisabledSTTChecks []string `json:"disabled_stt_checks"`
				// STT check intervals
				STTCheckIntervals models.STTCheckIntervals `json:"stt_check_intervals"`
			}{
				STTCheckIntervals: models.STTCheckIntervals{
					StandardInterval: 24 * time.Hour,
					RareInterval:     78 * time.Hour,
					FrequentInterval: 4 * time.Hour,
				},
			},
		}
```

Reply:
let's extract it and make non-embedded. let's keep code consistent.

Reply:
Done